apps/src/templates/sectionsRefresh/SectionsSetUpContainer.jsx

```diff
@@ -111,6 +127,51 @@ export default function SectionsSetUpContainer({sectionToBeEdited}) {
     setAdvancedSettingsOpen(!advancedSettingsOpen);
   };
 
+  const recordSectionSetupEvent = (e, section) => {
+    e.preventDefault();
```

Comment:
nit: is it worth moving this `preventDefault` down into the `onClick`? I could go either way on this so feel free to push back!

Reply:
Good call- refactored!